src/components/shared/basic/ReleaseProcess/AppPage/index.tsx

```diff
@@ -467,6 +472,37 @@ export default function AppPage() {
     }
   }
 
+  const handleDownload = async (
+    documentName: FileState[] | string,
+    documentId?: string
+  ) => {
+    let docId: string
+    let docName: string
+
+    if (Array.isArray(documentName)) {
```

Comment:
can you refactor it to a single statement?

ex - ```const documentName = Array.isArray(documentName) ? documentName[0].name : documentName```

Reply:
Changes done